internal/transfer_limits.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+)
+
+var (
+	SevenDayLimit = func() string {
+		if v := os.Getenv("TRANSFERS_SEVEN_DAY_SOFT_LIMIT"); v != "" {
+			return v
+		}
+		return "10000.00"
```

Comment:
Should we make these constants?

Reply:
Should we set these defaults high so that everything goes through for the normal user?

Reply:
We could make this a read-only function, but since it's an internal package I typically allow `var`'s because we can ensure it's never mutated. 

I wasn't sure if 10k / 25k were good limits. We can set the defaults to whatever.